templates/tutorialv2/view/container.html

```diff
@@ -266,8 +266,8 @@ <h2 class="force-blue">
              " class="ico-after more blue new-btn">
                 {% trans "Ajouter une section" %}
             </a>
-        {%  endif %}
-    {% elif version or version != content.sha_draf %}
```

Comment:
hum la condition était mauvaise mais pas sûr que l'enlever serait une bonne chose. `not version or  version != content.sha_draft` devrait aller mieux

Reply:
Vu que j'ai rien vu dans les logs de django, j'ai pas vu ça, je vais retester avec.